frontend/src/modules/editor/components/EditorProxy.test.js

```diff
@@ -24,4 +31,41 @@ describe('<EditorProxy>', () => {
 
         expect(wrapper.type()).toEqual(GenericEditor);
     });
+
+    it('resets failed checks when translation changes, but errors and warnings do not', () => {
+        const resetMock = sinon.stub();
+
+        const wrapper = mount(<EditorProxy
```

Comment:
I'm not sure you need to `mount` here, `shallow` might be sufficient. It's better to avoid using `mount` as much as possible, because `mount` will instantiate all the components in the hierarchy below this one, whether `shallow` only instantiate the one component you're testing, and no others.